docs/about/index.rst

```diff
@@ -8,6 +8,29 @@ About
 Mission
 -------
 
+Fairlearn is an open source toolkit that seeks to empower data scientists and
+developers to assess and improve fairness of their AI systems.
+The toolkit aspires to include:
+
+- Fairness metrics and unfairness mitigation algorithms
+- Interactive dashboard for fairness assessment
+- Educational materials (comprehensive user guide, detailed case studies,
+  white papers, etc.)
+
+Development of Fairlearn is firmly grounded in the understanding that fairness
+in AI systems is a sociotechnical challenge.  Because there are many complex
+sources of unfairness&mdash;some societal and some technical&mdash;it is not
+possible to fully "debias" a system or to guarantee fairness.
+Our goal is to enable humans to assess fairness-related harms, review the
+impacts of different mitigation strategies and then make trade-offs
+appropriate to their scenario.
+
+Fairlearn is a community-driven open source project, to be shaped through
+stakeholder engagement.  Its development and growth are guided by the belief
+that meaningful progress toward fairer AI systems requires input from a
+breadth of perspectives, ranging from data scientists, developers, and
+business decision makers to the people whose lives may be affected by the
+predictions of AI systems. 
```

Comment:
'of those AI systems' ?

Reply:
I think this applies to AI systems in general and adding "those" is potentially restricting in a way that we don't want. Also, do you feel like it adds something?

Reply:
I agree with @romanlutz -- I'd prefer to retain the original meaning.